audtorch/datasets/speech_commands.py

```diff
@@ -0,0 +1,115 @@
+import os
+import random
+
+from .utils import (download_url, extract_archive, safe_path)
+from .base import AudioDataset
+from .utils import safe_path
+
+
+class SpeechCommands(AudioDataset):
+    r"""An audio dataset of spoken words designed to help train and evaluate
+    keyword spotting systems
```

Comment:
Would be great to make the summary slightly shorter so it fits into [one line](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings).
Maybe: `Data set of spoken words designed for keyword spotting tasks.`

Reply:
Yeah that works. I just copied the first line off the abstract in the original paper, but I prefer your suggestion.